clang/docs/checkedc/rfc/Include-Checked-Hdrs.md

```diff
@@ -18,31 +18,29 @@ programs to include the checked counterparts of system header files either
 explicitly or implicitly during the conversion of these programs to Checked C.
 
 ## Problem Summary
-The existing approach for including checked header files, which is to explicitly
+The current approach for including checked header files, which is to explicitly
 replace an included header file with its checked counterpart, is inconvenient
-for automation by 3C. So we need a way to implicitly include checked header
+for adoption to Checked C - all the system header files included in a project
+need to be altered. So we need a way to implicitly include checked header
 files.
 
 ## Conditions that a Solution must Satisfy
- - The solution must not violate the "opt-in" philosophy that we advocate for
- the conversion of a program to Checked C. In accordance with this, the
- Checked C compiler must not silently perform an implicit inclusion of the
- checked header files.
- - The solution must be easy to integrate into a build system. 
- - The existing approach of explicitly including checked header files must
+ - The solution must provide users an option to either opt into the implicit
+ inclusion of checked header files, or opt out of it if checked header files
+ are implicitly included by default. This flexibility is required to support
+ custom build environments.
+ - The solution must be easy to integrate into an existing build system.
+ - The current approach of explicitly including checked header files must
  always remain a viable alternative.
- - It should be possible to directly include certain system header files and
- bypass the inclusion of their checked counterparts in order to support custom
- build environments.
  - The order in which the compiler driver searches the include paths must remain
  unaltered.
  - The solution must be able to accommodate any clang-specific declarations in
  system header files, if present (Ex.: inttypes.h).
- - The Checked-C-specific declarations must live in checkedc repository.
+ - The Checked-C-specific declarations must live in the checkedc repository.
```

Comment:
`Checked-C-specific` --> `Checked C-specific`

Reply:
Thank you for your review! I feel that `Checked-C-specific` is a bit more clear - so I have retained the same.